### PR TITLE
Add a leave policy for governing board members 🏖️

### PR DESCRIPTION
**Pull request by bobcatfish**
Since I am soon going to be on leave, I brought this up in a recent
governing board / community meeting, and this commit adds the policy
that we discussed adding for governing board members going on leave in
general. We wanted to make sure that this option was available
regardless of the reason for the leave.

The maximum period of time for the leave is however conveniently the
exact length of time I plan to be gone XD , so if folks have different
opinions on what the threshold should be, we can definitely change it

### CODE REVIEW

**Comment by bobcatfish** [comment]
PTAL @afrittoli @vdemeester @dibyom @priyawadhwa  - I also put this [on the agenda for our next meeting](https://docs.google.com/document/d/1g2HQk_4ypYEWPX2WjOFa6vOMpwYoRNprph_P09TS1UM/edit#heading=h.ej1ypg4j8j6v)

**Review by priyawadhwa — APPROVED** [review]
sgtm!

**Review by dibyom — APPROVED** [review]

File: governance.md [hunk 1]
```diff
@@ -302,3 +311,19 @@ Members of the governing board will be given access to these resources:
...
+Any member of the governing board is welcome to take leave from being a governing board member, for any reason (which
+they are not required to disclose). The actions taken will depend on both of:
+
+1. The length of the leave: whether this is greater or less than 1/4 of their total term (since terms are 2 years, this
+   means the criteria is whether or not the leave is greater than 6 months).
+2. Whether their seat will be up for election during the period of their leave
+
+If the leave is less than or equal to 1/4 of the board member's total term, and their seat will not be up for
+re-election during that time, the governing board member may select someone to be an acting governing board member for
+the length of the leave (while maintaining the [maximum representation requirement](#maximum-representation)).
```
Comment (afrittoli) [inline governance.md:326]: Should the acting governing board member also fulfil the [candidate  eligibility](https://github.com/tektoncd/community/blob/main/governance.md#candidate-eligibility) ?

**Review by afrittoli — COMMENTED** [review]
Thank you @bobcatfish - just one comment.

**Comment by bobcatfish** [comment]
Ready for another look @afrittoli @vdemeester

**Review by afrittoli — COMMENTED** [review]
Thanks for the updates, lgtm

**Review by vdemeester — APPROVED** [review]
/meow

**Pull request merged**